http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -242,10 +242,17 @@ object clusterQuery extends TableQuery(new ClusterTable(_)) {
 
   // select * from cluster c
   // join cluster_image ci on c.id = ce.cluster_id
-  val clusterJoinClusterImageQuery: Query[(ClusterTable, ClusterImageTable), (ClusterRecord, ClusterImageRecord), Seq] =
+  // left join label l on c.id = l.clusterId
+  val clusterMetricsQuery: Query[(ClusterTable, ClusterImageTable, Rep[Option[LabelTable]]),
+                                 (ClusterRecord, ClusterImageRecord, Option[LabelRecord]),
+                                 Seq
+  ] =
     for {
-      (cluster, image) <- clusterQuery join clusterImageQuery on (_.id === _.clusterId)
-    } yield (cluster, image)
+      ((cluster, image), label) <-
+        clusterQuery join clusterImageQuery on (_.id === _.clusterId) joinLeft labelQuery on { case ((c, _), lbl) =>
```

Comment:
Hopefully joining runtimes to labels in the metrics query is okay.. we use the labels for distinguishing Terra vs AoU runtimes, which is useful info. It seemed to perform okay on dev.

Reply:
Makes sense. Yeah I think the tables are indexed correctly so it should be okay.